From 4b9efc24b12ff2ea49d3c1c8c774ba6e158a8779 Mon Sep 17 00:00:00 2001
From: Caesar Wang <wxt@rock-chips.com>
Date: Mon, 10 Aug 2020 17:04:32 +0800
Subject: [PATCH] arm64: rockchip_linux_defconfig: remove the bifrost

This driver is used by rk3326_linux_defconfig or px30_linux_defconfig

Signed-off-by: Caesar Wang <wxt@rock-chips.com>
Change-Id: I58137f3dbe58b2442674331f3ec554c825ff625a
---
 arch/arm64/configs/rockchip_linux_defconfig | 8 +-------
 1 file changed, 1 insertion(+), 7 deletions(-)

diff --git a/arch/arm64/configs/rockchip_linux_defconfig b/arch/arm64/configs/rockchip_linux_defconfig
index da31f5c8d48d..11d03f981143 100644
--- a/arch/arm64/configs/rockchip_linux_defconfig
+++ b/arch/arm64/configs/rockchip_linux_defconfig
@@ -305,18 +305,12 @@ CONFIG_MALI450=y
 CONFIG_MALI_SHARED_INTERRUPTS=y
 CONFIG_MALI_DT=y
 CONFIG_MALI_DEVFREQ=y
-CONFIG_MALI_MIDGARD=m
+CONFIG_MALI_MIDGARD=y
 CONFIG_MALI_EXPERT=y
 CONFIG_MALI_PLATFORM_THIRDPARTY=y
 CONFIG_MALI_PLATFORM_THIRDPARTY_NAME="rk"
 CONFIG_MALI_DEBUG=y
 CONFIG_MALI_PWRSOFT_765=y
-CONFIG_MALI_BIFROST_FOR_LINUX=y
-CONFIG_MALI_BIFROST=y
-CONFIG_MALI_BIFROST_DEVFREQ=y
-CONFIG_MALI_PLATFORM_NAME="rk"
-CONFIG_MALI_BIFROST_EXPERT=y
-CONFIG_MALI_BIFROST_DEBUG=y
 CONFIG_BACKLIGHT_LCD_SUPPORT=y
 # CONFIG_LCD_CLASS_DEVICE is not set
 CONFIG_BACKLIGHT_CLASS_DEVICE=y
-- 
2.35.3

